sources/ClangSharp/Generated/CXCursor.cs

```diff
@@ -1,11 +1,31 @@
-using System;
-
 namespace ClangSharp
 {
     public partial struct CXCursor
     {
+        [NativeTypeName("enum CXCursorKind")]
         public CXCursorKind kind;
+
         public int xdata;
-        public IntPtr data0; public IntPtr data1; public IntPtr data2;
+
+        [NativeTypeName("const void *[3]")]
+        public _data_e__FixedBuffer data;
```

Comment:
I find the attributes to, at a glance, make it much easier to determine the original signature. While still allowing you to use the least expensive managed signature. Here is a good example.